### PR TITLE
topology: adding optional support for IIR component on jsl-rt015 topology

### PR DESCRIPTION
**Pull request by rosschisholm-xperi**
Allows the IIR component to be added to the jsl-rt015 topology allowing tuning

Signed-off-by: Ross Chisholm <ross.chisholm@xperi.com>

### CODE REVIEW

File: tools/topology/sof-jsl-rt5682.m4 [hunk 1]
```diff
@@ -62,12 +62,24 @@ PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
 	1000, 0, 0,
 	48000, 48000, 48000)
 
+ifdef(`INCLUDE_IIR_EQ',
+`
 # Low Latency playback pipeline 2 on PCM 1 using max 2 channels of s24le.
```
Comment (cujomalainey) [inline tools/topology/sof-jsl-rt5682.m4:67]: this comment is wrong, mind updating it?

**Review by cujomalainey — APPROVED** [review]

**Comment by cujomalainey** [comment]
CI failures unrelated

File: tools/topology/sof-jsl-rt5682.m4 (hunk 1 above)
Comment (rosschisholm-xperi) [reply tools/topology/sof-jsl-rt5682.m4]: all done!

**Review by cujomalainey — APPROVED** [review]
thanks for the fix

**Review by lgirdwood — COMMENTED** [review]
LGTM, just the comment

**Review by lgirdwood — APPROVED** [review]
Ignore my last response, I had looked at the older version.

**Pull request merged**